domain/src/main/java/com/harman/roomdbapp/domain/repository/RandomNumberRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.harman.roomdbapp.domain.repository
+
+import androidx.lifecycle.LiveData
+import com.harman.roomdbapp.domain.model.RandomNumber
+
+interface RandomNumberRepository {
+
+    suspend fun addNumber(number: RandomNumber)
+
+    suspend fun getNumbers(): LiveData<List<RandomNumber>>
```

Comment:
as described above, please remove `suspend` from methods, which return observable-like object. Also here please replace `LiveData` to `Flow`